kubernetes/linera-validator/build_and_redeploy.sh

```diff
@@ -141,8 +141,12 @@ kubectl get pods
 echo -e "\nServices:"
 kubectl get svc
 
-docker rm linera-test-local || true
-docker run -d --name linera-test-local "$docker_image"
```

Comment:
Are you removing this stuff on purpose?

Reply:
Yes, forgot to put a comment here clarifying. This was needed when we were copying stuff from the docker container with `docker cp`. We're not doing that anymore, so it's not needed anymore